### ISSUE TITLE
UPDATE?

### ISSUE THREAD

**Issue by gameb0y** [andryou/scriptsafe#128]
ANDRY, PLEASE BACK! 

NO UPDATE FOR MONTHS :/

**Comment by snowbound** [comment]
As I mentioned before I switched over to another extension. Sorry to say it does not offer the granularity as scriptsafe but I am not seeing the issues either.

**Comment by AykutCevik** [comment]
@snowbound can you tell us which extension? And can you tell us under which license you contributed scriptsafe?

**Comment by snowbound** [comment]
I am using uMatrix from the developer of uBlock Origin but its a bit frustrating at times. I also in the past used ScriptBlock both under Chrome.

**And can you tell us under which license you contributed scriptsafe?**

Sorry but I was simply an end user of ScriptSafe and was not involved with its coding.

**Comment by AykutCevik** [comment]
@snowbound sorry, I misunderstood that. 

So we need a message from @andryou.

**Comment by peterlaws** [comment]
Where is @andryou anyway? Family problems maybe, health problems?

Be good to know as I found an issue, you can not scroll the pop-up by the arrows, you need to drag the actual scrollbar

**Comment by SpiralBrad** [comment]
He's active on Twitter, and ignoring messages regarding this project.

**Comment by AykutCevik** [comment]
@SpiralBrad I already tried to connect with him on Twitter but he didn't answer my request :/

**Comment by andryou** [comment]
Hi all, I'm still alive! My apologies for being MIA; I will be going through the reports over the next couple of weeks. I do have some updates to commit, and will go through the reports.

Thank you @AykutCevik for your pull request, I had a quick look and I like how you were able to work with my code :) Will take a closer look soon.

**Comment by AykutCevik** [comment]
@andryou nice to see that you are back again and still alive ;)

**Comment by andryou** [comment]
Just to follow-up on this issue: thank you all and my apologies again for my absence.

uMatrix is a great extension and I have an incredible amount of respect for Gorhill.

@peterlaws I have noticed that the panel is only scrollable by the scrollbar occasionally. I believe this may be a new bug in Chrome itself. In working on v1.0.8.5 I did try to address this (by setting the height of the panel to 100% and to a pixel value to see if this helped, but no luck).

Just to speak on ScriptSafe's license, as of v1.0.8.5 (coming out by end of this week), it will be under the GNU GPL v3 license.
